ci/deps/azure-37-numpydev.yaml

```diff
@@ -15,6 +14,7 @@ dependencies:
   - pytz
   - pip
   - pip:
+    - cython>=0.29.16
```

Comment:
Why did some of these get moved down to `pip`?

Reply:
This builds don't use the conda-forge channel and so (for some reason) don't have 0.29.16.  We have to fall back to using pip.